src/OSPSuite.Core/Services/EntitySourcePathUpdater.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using OSPSuite.Core.Domain;
+using OSPSuite.Core.Domain.Builder;
+using OSPSuite.Core.Domain.Services;
+using OSPSuite.Utility.Extensions;
+
+namespace OSPSuite.Core.Services
+{
+   public interface IEntitySourcePathUpdater
+   {
+      void UpdateEntityPath(IModel model, SimulationBuilder simulationBuilder);
```

Comment:
I would make this `UpdateEntityPaths` with 's'

Reply:
ok